bottery/telegram/engine.py

```diff
@@ -85,6 +85,13 @@ def __init__(self, *args, **kwargs):
 
         updates = await self.api.get_updates(**payload)
 
+        if not updates.get('ok'):
+            msg = '[{}] Unable to receive updates, data={}, error'.format(
+                self.platform,
+                updates
+            )
+            raise Exception(msg)
```

Comment:
Should we create a specific exception? Something like `PlatformError`? What do you think?

Reply:
Sure, I'll change an `Exception` to something more meaningful 😄